docs/integrations/microsoft-azure/azure-storage.md

```diff
@@ -60,6 +61,48 @@ When you configure the event hubs source or HTTP source, plan your source catego
 
 Metrics and logs in Azure Monitor support only Azure Resource Manager storage accounts. Azure Monitor doesn't support classic storage accounts. If you want to use metrics or logs on a classic storage account, you need to migrate to an Azure Resource Manager storage account. For more information, see [Migrate to Azure Resource Manager](https://learn.microsoft.com/en-us/azure/virtual-machines/migration-classic-resource-manager-overview).
 
+## Configure Field in Field Schema
+1. In the main Sumo Logic menu, select **Manage Data** > **Logs** > **Fields**.
+2. Search for the `functionname` field.
```

Comment:
we are not using functionname field in azure, add the azure fields you have simply copy pasted the lambda docs, that doc was just for reference.